packages/owl-bot/src/bin/commands/commit-post-processor-update.ts

```diff
@@ -45,6 +45,12 @@ export async function commitPostProcessorUpdate(repoDir = ''): Promise<void> {
   }
   // Add all pending changes to the commit.
   cmd('git add -A .', {cwd: repoDir});
+  const status = cmd('git status --porcelain', {cwd: repoDir}).toString(
```

Comment:
might be worth adding a comment that `// git status --porcelain returns empty string if no local changes`.

I'm going ahead and merging, since it seems like this bug would be pretty serious for `.NET` folks.

Reply:
Thank you for merging.  Comment added in https://github.com/googleapis/repo-automation-bots/pull/2666.